beancount_import/source/schwab_csv.py

```diff
@@ -1098,7 +1104,7 @@ def __init__(self, journal: JournalEditor, lots: LotsDB) -> None:
         self.found_accounts: Set[str] = set()
 
     def process_entry(self, raw_entry: RawEntry) -> Optional[TransactionEntry]:
-        account = self.schwab_to_account.get(raw_entry.account)
+        account = self.schwab_to_account.get(raw_entry.account) or self.schwab_to_account.get(raw_entry.account[-9:]) # try only the XXXX-1234 part
```

Comment:
Do you know for sure that this "initial part" is always exactly 9 characters long? What is the "initial part" in your case? Can we have a more robust way to identify the real account number and strip off the rest? Ideally where we construct the account map, not here...

Reply:
I think I did it here because in some files it is "XXXX-1234" (length 9), in some files it is like "IRA XXX-1234". We can leave this part to another time.

Reply:
I filed #122 for follow-up on this.